cpg_infra/config/config.py

```diff
@@ -123,6 +123,13 @@ class GCP(DeserializableDataclass):
         gcp: GCP
         slack_channel: str | None = None
         etl_accessors: list[str] = dataclasses.field(default_factory=list)
+        # Metamist environment (DEVELOPMENT / PRODUCTION) for ETL cloud functions
+        etl_environment: str | None = None
```

Comment:
Can you provide 'PRODUCTION' as an explicit default here?

Reply:
No worries, changed that default.